### PR TITLE
Feature/11 pop up cont

### PR DESCRIPTION
**Pull request by bryanakitchen**
Updated pop ups and reset option which resets the increment of time. Currently hardcoded messages to come up every 10 minutes for the sake of testing publishing.

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -22,55 +21,50 @@ const activate = async (context) => {
...
 
-		 return resetTimer
-    });
-
-	    // THIS IS WHERE STOP WATCH USER COMMAND FUNCTIONS BEGIN
+			if (userChoice === 'Move your body') {
+				const randomLink = await fetch.get('http://be-human-demo-staging.herokuapp.com/api/v1/links/random')
+				vscode.env.openExternal(vscode.Uri.parse(`${randomLink.body.url}`));
+			}
+		}, 60000)
```
Comment (ShaneUP1) [inline extension.js]: I think 10min would actually be 600,000s
1s = 1000ms * 60s = 1min * 10min = 600,000

**Review by ShaneUP1 — APPROVED** [review]
Left a comment on setInterval time...I think 10min is actually 600,000ms

**Review by jillianlg — APPROVED** [review]
This looks great Bryana. Just double check the math in Shanes commit and you should be good to go!

**Review by bryanakitchen — COMMENTED** [review]
Ok, Shane, I think I've made that change to 10 minutes, for real. :)

**Review by jillianlg — APPROVED** [review]
Looks good.

**Pull request merged**